cmd/create/cluster/cmd.go

```diff
@@ -996,6 +991,18 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	if domainPrefix == "" && len(clusterName) > ocm.MaxClusterDomainPrefixLength {
```

Comment:
looks like this made codecov unhappy. I'll try to update the tests

Reply:
Extracted this logic onto something that I can test. Let's see if that will make CI happy.

Reply:
done